README.md

```diff
@@ -36,8 +36,8 @@ NEAR_ACCOUNT_ID="[NEAR_TESTNET_ACCOUNT]"
 NEAR_PRIVATE_KEY="[NEAR_ACCOUNT_PRIVATE_KEY]"
 MPC_PATH="[MPC_PATH]"
 MPC_CHAIN="[ethereum|bitcoin]"
-MPC_CONTRACT_ID="v1.signer-dev.testnet"
-MPC_PUBLIC_KEY="secp256k1:54hU5wcCmVUPFWLDALXMh1fFToZsVXrx9BbTbHzSfQq1Kd1rJZi52iPa4QQxo6s5TgjWqgpY8HamYuUDzG6fAaUq"
+MPC_CONTRACT_ID="multichain-testnet-2.testnet"
```

Comment:
I don't think this is the latest contract.

We're currently using `v1.signer-dev.testnet` AFAIK.